### DISCUSSION TITLE
KeePassXC 2.7.9 doesn't start

### DISCUSSION THREAD

**Discussion by M-S3vN** [keepassxreboot/keepassxc#11205] [Q&A] [answered]
I have KeePassXC installed on 3 Win10 PCs
#1: installed 2.7.9 directly -> works
#2: installed 2.7.7 -> updated to 2.7.9 -> works
#3: installed 2.7.7 -> works
but if I update #3 to 2.7.9 -> KeePassXC will not start. If I unistall it and reinstall 2.7.9 it still does not work. If I unistall it and install 2.7.7 it works

**Comment by droidmonkey** [comment, marked as answer]
https://github.com/keepassxreboot/keepassxc/issues/10925
